.pipelines/templates/run-upstream-e2e.yaml

```diff
@@ -0,0 +1,12 @@
+steps: 
+  - bash: |
+      git clone -b ${UPSTREAM_BRANCH} $(upstream.repo) $(repo.path)/osm
```

Comment:
Why is it ` $(repo.path)/osm` here but ` $(repo.path)` elsewhere?

Reply:
It was not working without /osm before because I was trying to clone multiple repositories to $(repo.path)